operator-pipeline-images/operatorcert/schemas/ci-schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "title": "ci.yaml validation schema",
+    "description": "json schema validating the ci.yaml",
+    "type": "object",
+    "additionaProperties": true,
+    "properties": {
```

Comment:
There are couple of other properties that we use from the ci file. 

- updateGraph (str, optional)
- reviewers (list of strings, optional)
- merge (bool, optional)
- cert_project_id (str, optional)

Please add the missing properties to the schema.

Reply:
Updated. As I was adding these fields, I realized that the `updateGraph` property is also valid for enum as we need to have only two values `semver-mode` or `replaces-mode` using for that. Should I add the enum for this property?

Reply:
The place where you added the additional fields is not correct. You put them under `fbc` object however they are on the root level of the file.

Reply:
Let's keep it simple for now and don't add any additional enums.